orquesta/conducting.py

```diff
@@ -634,6 +637,39 @@ def _evaluate_task_actions(self, task):
 
         return task
 
+    def has_barrier_next(self, task_id, route=None):
```

Comment:
This method has a lot of duplicated code with the has_next_tasks, is it possible to put some of it into a common method?

Reply:
Done